Block/Adminhtml/Report/Edit/Tabs.php

```diff
@@ -105,18 +105,18 @@ protected function _beforeToHtml()
             ->getFirstItem()
             ->getData('id');
         // only show tab if default account_id and application_id exist
-        if ($account_id and $application_id) {
-            $this->addTab(
-                'report_edit_tabs_metrics',
-                [
-                    'label' => __('View Site Metrics'),
-                    'title' => __('View Site Metrics'),
-                    'content' => $this->getLayout()->createBlock(
-                        'Sectionio\Metrics\Block\Adminhtml\Report\Edit\Tab\Metrics'
-                    )->toHtml()
-                ]
-            );
-        }
+        // if ($account_id and $application_id) {
+        //     $this->addTab(
+        //         'report_edit_tabs_metrics',
+        //         [
```

Comment:
Remove instead of comment out? Unless it's coming back in very soon. We'll always have git history if we need the code back.